<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelsGenWin</class>
 <widget class="QWidget" name="CModelsGenWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModelsGen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>68</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>68</height>
      </size>
     </property>
     <property name="title">
      <string>数据库设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="4">
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>用户名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLineEdit" name="m_iDatabaseNameEdit">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="m_iPortSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>3306</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>端口号:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>数据库名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="m_iPasswdEdit">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="m_iUsernameEdit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_iHostEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QPushButton" name="m_iOpenBtn">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="m_iTabGBox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="m_iSearchEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_iSearchBtn">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CPageWin" name="m_iPageWin" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="m_iShowGBox">
     <property name="title">
      <string>显示</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>表前缀:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_iTabPrefixEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>t_</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="m_iShowSqlRBtn">
          <property name="text">
           <string>显示建表sql</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_iShowModelRBtn">
          <property name="text">
           <string>显示model代码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_iExportModelCodeBtn">
          <property name="text">
           <string>导出model代码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="m_iShowEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPageWin</class>
   <extends>QWidget</extends>
   <header>page_win.h</header>
   <container>1</container>
   <slots>
    <signal>PageChangeRequest(int32_t)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_iDatabaseNameEdit</tabstop>
  <tabstop>m_iPortSpinBox</tabstop>
  <tabstop>m_iPasswdEdit</tabstop>
  <tabstop>m_iUsernameEdit</tabstop>
  <tabstop>m_iHostEdit</tabstop>
  <tabstop>m_iOpenBtn</tabstop>
  <tabstop>m_iShowSqlRBtn</tabstop>
  <tabstop>m_iShowModelRBtn</tabstop>
  <tabstop>m_iSearchEdit</tabstop>
  <tabstop>m_iSearchBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_iOpenBtn</sender>
   <signal>clicked()</signal>
   <receiver>CModelsGenWin</receiver>
   <slot>__OpenDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>956</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>976</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iSearchBtn</sender>
   <signal>clicked()</signal>
   <receiver>CModelsGenWin</receiver>
   <slot>__OnSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iShowSqlRBtn</sender>
   <signal>clicked()</signal>
   <receiver>CModelsGenWin</receiver>
   <slot>__OnShowRBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>977</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iShowModelRBtn</sender>
   <signal>clicked()</signal>
   <receiver>CModelsGenWin</receiver>
   <slot>__OnShowRBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>976</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iExportModelCodeBtn</sender>
   <signal>clicked()</signal>
   <receiver>CModelsGenWin</receiver>
   <slot>__OnExportModelCode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>833</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>979</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>__OpenDB()</slot>
  <slot>__OnSearch()</slot>
  <slot>__OnTabSelected(QModelIndex)</slot>
  <slot>__OnShowRBtnClicked()</slot>
  <slot>__OnExportModelCode()</slot>
 </slots>
</ui>
